InvenTree/part/models.py

```diff
@@ -3295,6 +3296,9 @@ class PartParameterTemplate(MetadataMixin, models.Model):
     Attributes:
         name: The name (key) of the Parameter [string]
         units: The units of the Parameter [string]
+        description: Description of the parameter [string]
+        type: The data type of the Parameter [integer]
+        choices: A comma-separated list of choices for the Parameter [string]
```

Comment:
Question: Where are type / choices defined?

Reply:
@matmair I'm going to move this into https://github.com/inventree/InvenTree/issues/4835